behavioral-questions.md

```diff
@@ -4,16 +4,16 @@ Answer the following questions. Use the Nugget + STAR method for each answer.
 
 1. Why did you choose a career in software development?
 
-- Nugget:
-- Situation:
-- Task:
-- Action:
-- Result:
+- Nugget: I spent approximately four years in the Navy as a weather forecaster.
+- Situation: Eventually, there came a point where it was apparent I would not be able to continue my service in the military.
+- Task: My superiors were adamant about me having a plan before I separated. They wanted to know what I planned to do once I left.
+- Action: I talked to my superiors, my family, and my friends. They each gave me suggestions on what I should do, but it was an old college friend of mine that encouraged me to become a programmer.
+- Result: I thought VETTEC would provide me with a solid opportunity to try programming without a heavy personal financial investment or commitment of time. LEARN Academy was my first choice and I'm grateful to have been able to attend their cohort.
 
 2. In previous roles, how have you helped teammates succeed?
 
-- Nugget:
-- Situation:
-- Task:
-- Action:
-- Result:
\ No newline at end of file
+- Nugget: In the Navy, I spent two years in Norfolk, VA as an assistant weather forecaster.
+- Situation: I was fortunate to have been surrounded by many wonderful people, from the officers, to the civilians, to my fellow peers.
+- Task: As an assistant, I believed it was my duty, first and foremost, to learn as much as I could before I left for C-School. I needed to become competent and dependable. But I also needed to be a good teammate.
+- Action: And part of being a good teammate is allowing others to have the opportunity to grow and show their own value. Sometimes that meant sharing what I had learned and training others. Sometimes that meant putting those under me in roles they might not want, but would challenge them and help the team. Sometimes that meant taking on a role I may not want, but would allow other teammates to focus on pursuing other roles and career milestones. And sometimes, it just meant trusting my teammates and staying out of their way.
+- Result: I do not know how beneficial any of this actually was. I was just trying to do the best I could.
```

Comment:
For this answer, I think focusing on one specific scenario that you may have a measurable result could make this answer stand out.  Also, consider modifying the verbiage "those under me" to "those I supervise" while focusing on the positive outcome of how you gave them a chance to grow.